keyboards/handwired/reclined/keymaps/pdl/rules.mk

```diff
@@ -0,0 +1 @@
+USER_NAME = pdl
```

Comment:
Just a note, if your keymap name matches your userspace name you don't need to set this rule.

Reply:
```suggestion
```